### PR TITLE
Showing wrong recorder time text

### PR DESCRIPTION
**Pull request by Ajmal-M-A**
Showing wrong recorder time text when device timezone is different ( In minutes).

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -45,8 +46,11 @@ class _MyAppState extends State<MyApp> {
       print('startRecorder: $path');
 
       _recorderSubscription = flutterSound.onRecorderStateChanged.listen((e) {
-        DateTime date = new DateTime.fromMillisecondsSinceEpoch(e.currentPosition.toInt());
-        String txt = DateFormat('mm:ss:SS', 'en_US').format(date);
+        initializeDateFormatting();
```
Comment (hyochan) [inline example/lib/main.dart]: Does this should be called in every callback? What if just initializing this once?

File: example/lib/main.dart [hunk 2]
```diff
@@ -45,8 +46,11 @@ class _MyAppState extends State<MyApp> {
       print('startRecorder: $path');
```
Comment (hyochan) [inline example/lib/main.dart]: Hope you can remove this.

File: example/lib/main.dart (hunk 1 above)
Comment (Ajmal-M-A) [reply example/lib/main.dart]: Hi Hyo Chan, Not required in every callback. Moved the initialization statement to init.

File: example/lib/main.dart (hunk 2 above)
Comment (Ajmal-M-A) [reply example/lib/main.dart]: This line already exist in the source, Hopes it will be helpful for the debug purpose. So not removed.

**Pull request merged**